apisix/plugins/ldap-auth.lua

```diff
@@ -139,8 +141,20 @@ function _M.rewrite(conf, ctx)
     local uid = conf.uid or "cn"
 
     local userdn =  uid .. "=" .. user.username .. "," .. conf.base_dn
-    local ld = lualdap.open_simple (conf.ldap_uri, userdn, user.password, conf.use_tls)
-    if not ld then
+    local ldapconf = {
+        timeout = 10000,
+        start_tls = false,
+        ldap_host = conf.ldap_host,
+        ldap_port = conf.ldap_port,
+        ldaps = conf.use_tls,
+        verify_ldap_host = conf.verify_ldap_host,
+        base_dn = conf.base_dn,
+        attribute = uid,
+        keepalive = 60000,
```

Comment:
Let's doc the newly added timeout and keepalive

Reply:
These items are not exported to admin api, we only use default values. So we need to doc the default values? But it's kind of internal parameters.